File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/elixir/tinix.ex
defmodule FizzBuzz do
  def fizzbuzz(n) when rem(n, 3) == 0 and rem(n, 5) == 0, do: "fizzbuzz"
  def fizzbuzz(n) when rem(n, 3) == 0, do: "fizz"
  def fizzbuzz(n) when rem(n, 5) == 0, do: "buzz"
  def fizzbuzz(n), do: Integer.to_string(n)
end

1..100
  |> Enum.map(&FizzBuzz.fizzbuzz/1)
  |> Enum.each(&IO.puts/1)

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/ruby/tinix.rb

def fizzbuzz(range)
  (1..range).each do |number|
    if (number % 3).zero? && (number % 5).zero?
      puts 'FizzBuzz'
    elsif (number % 3).zero?
      puts 'fizz'
    elsif (number % 5).zero?
      puts 'buzz'
    else
      puts number
    end
  end
end

fizzbuzz(100)
